Sources/AardvarkLoggingUI/Log Viewing/ARKLogTableViewController.m

```diff
@@ -126,61 +125,18 @@ - (void)viewDidLoad;
     self.tableView.rowHeight = 34.0;
     self.tableView.dataSource = self;
     self.tableView.delegate = self;
-    
-    if ([UIDevice currentDevice].systemVersion.floatValue >= 8.0) {
-        self.searchController = [[UISearchController alloc] initWithSearchResultsController:nil];
-        self.searchController.searchResultsUpdater = self;
-        self.searchController.delegate = self;
-        self.searchController.dimsBackgroundDuringPresentation = NO;
-        self.tableView.tableHeaderView = self.searchController.searchBar;
-    }
+
+    self.searchController = [[UISearchController alloc] initWithSearchResultsController:nil];
+    self.searchController.searchResultsUpdater = self;
+    self.searchController.delegate = self;
+    self.searchController.obscuresBackgroundDuringPresentation = NO;
+    self.tableView.tableHeaderView = self.searchController.searchBar;
 
     if (self.title.length == 0) {
         self.title = NSLocalizedString(@"Logs", @"Title of log viewing screen.");
     }
 }
 
-#pragma mark - UIActionSheetDelegate
-
-- (void)actionSheet:(UIActionSheet *)actionSheet clickedButtonAtIndex:(NSInteger)buttonIndex;
-{
-#if TARGET_IPHONE_SIMULATOR
```

Comment:
looks like this code block was moved down to `_exportLogsWithOption` with no meaningful changes